config/samples/xjoin_v1alpha1_xjoinindex.yaml

```diff
@@ -39,5 +39,12 @@ spec:
           "transformation.parameters": {
             "delimiters": ["/", "="]
           }
+        }, {
```

Comment:
this is part of the new xjoin version. You can omit this, I'll find another way to do it for the next iteration.